gridlod/lod.py

```diff
@@ -495,13 +591,21 @@ def computeRhsCoarseQuantities(patch, corrector, aPatch):
     TInd = util.convertpCoordIndexToLinearIndex(patch.NPatchCoarse-1, patch.iElementPatchCoarse)
     
     Rmsi = np.zeros(NpPatchCoarse)
+
+    NPatchCoarse = patch.NPatchCoarse
+    NTPrime = np.prod(NPatchCoarse)
+    cetaTPrime = np.zeros(NTPrime)
     
     def accumulate(TPrimeInd, TPrimei, P, Q, KTPrime, BTPrimeij, CTPrimeij):
         Rmsi[TPrimei] = BTPrimeij[:,0]
+        cetaTPrime[TPrimeInd] = CTPrimeij       # < -- This is new !! is that correct ?
```

Comment:
Yes, I think it is correct. It should correspond exactly to  ||A_ref^1/2 nabla R_ref f_ref ||^2